### PR TITLE
apply run ecs check-heredoc-nowdoc command to packages and rules directory

### PR DESCRIPTION
**Pull request by samsonasik**
Run `ecs` command for `HeredocNowdocPHPCodeFormatter` to `rules` and `packages` directory:

```
packages/easy-coding-standard/bin/ecs check-heredoc-nowdoc ../rector/packages
packages/easy-coding-standard/bin/ecs check-heredoc-nowdoc ../rector/rules
```

by https://github.com/symplify/symplify/pull/2137

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Thanks for this :+1: 

I wonder how can we get rid of `strict_types`, because they often seem as unnecesary clutter.

We could have standalone `ecs.php`, but that seems as extra work. If we apply "keep the same as before" approach, it might be hard to add these types.

What do you think?

**Comment by samsonasik** [comment]
what about add option like `--no-strict-declaration` option?

**Comment by TomasVotruba** [comment]
Sounds good :+1:

**Comment by samsonasik** [comment]
I've added `--no-strict-types-declaration` option at https://github.com/symplify/symplify/pull/2141 and run on rules and packages directory:

```
packages/easy-coding-standard/bin/ecs check-heredoc-nowdoc ../rector/packages --no-strict-types-declaration
packages/easy-coding-standard/bin/ecs check-heredoc-nowdoc ../rector/rules --no-strict-types-declaration
```

File: rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php [hunk 1]
```diff
@@ -33,9 +33,9 @@ public function getDefinition(): RectorDefinition
                 new CodeSample(
                     '$this->setExpectedException(Exception::class, "Message", "CODE");',
                     <<<'PHP'
-$this->setExpectedException(Exception::class);
-$this->expectExceptionMessage("Message");
-$this->expectExceptionCode("CODE");
+$this->setExpectedException(Throwable::class);
```
Comment (samsonasik) [inline rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php:36]: I am not sure why the ecs change `Exception::class` to `Throwable::class` on this, @TomasVotruba could you verify?

Comment (TomasVotruba) [reply rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php:36]: There is a rule that does this change as part of upgrade to PHP 7.0.

Could you update the line above also to heredoc?
That way the change would be in both snippets and correct

Comment (samsonasik) [reply rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php:36]: I've updated to heredoc

Comment (samsonasik) [reply rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php:36]: Do you think `Throwable::class` need to be reverted to `Exception::class` ?

Comment (samsonasik) [reply rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php]: I reverted to nowdoc and changed back to `Exception::class`

Comment (samsonasik) [reply rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php]: Now all green, I reverted to nowdoc as phpstan complain about it.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**